exporter/trace/stdout/stdout.go

```diff
@@ -25,6 +25,9 @@ import (
 
 // Options are the options to be used when initializing a stdout export.
 type Options struct {
+	// File is the destination.  If not set, os.Stdout is used.
+	File io.Writer
```

Comment:
Prefer using `Destination` given it doesn't have to be a file; otherwise people may assume incorrectly it _has_ to be on the filesystem.

Reply:
In the statsd config, it's called "Writer", e.g.,
```
		// Writer is an alternate to providing a URL.  When Writer is
		// non-nil, URL will be ignored and the exporter will write to
		// the configured Writer interface.
		Writer io.Writer
```

Reply:
I'm fine with both naming, however the original one is from  exporter/metric/stdout/stdout.go.
We may need to also rename the `File` field in Options?

Reply:
Yes, let's do that in the `Options` across the board.